src/sentry/tagstore/snuba/backend.py

```diff
@@ -218,7 +218,7 @@ def __get_tag_keys(
         limit=1000,
         keys=None,
         include_values_seen=True,
-        include_transactions=False,
```

Comment:
Here and on line 428 of this file, we set the `dataset` default to be Events (instead of Discover), since `include_transaction`'s default was False